### PR TITLE
[INF154, 203]: tinyplugins support for mathjax and preview issues

### PR DESCRIPTION
**Pull request by mehaknasir**
Ticket Links: https://2u-internal.atlassian.net/browse/INF-54, https://2u-internal.atlassian.net/browse/INF-203

This PR contains the following features

- Add support for emojis in tinyMCE
- Add support for mathjax in TinyMCE
- Fix the Preview issue with react-mathjax-preview package for both new and legacy experience
- dropped usage of rendered_body and preview_body fields and only used raw_body as the only raw body can provide actual data
- a fix was needed to restrict the height of the post summary card as in the case of MathML the text was not truncating.
- timeout increased for jest test cases to remove inconsistent behavior of failing of test cases
Screenshot 2022-05-11 at 5 19 32 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "@edx/paragon": "19.10.1",
     "@reduxjs/toolkit": "1.8.0",
     "@tinymce/tinymce-react": "3.13.1",
+    "@wiris/mathtype-tinymce5": "^7.28.0",
```
Comment (awais-ansari) [inline package.json:43]: nit: use specific version
"@wiris/mathtype-tinymce5": "^7.28.0" => "@wiris/mathtype-tinymce5": "7.28.0"

File: package.json [hunk 2]
```diff
@@ -49,6 +50,7 @@
     "raw-loader": "4.0.2",
     "react": "16.14.0",
     "react-dom": "16.14.0",
+    "react-mathjax-preview": "^2.2.6",
```
Comment (awais-ansari) [inline package.json:53]: nit: use a specific version
"react-mathjax-preview": "^2.2.6" => "react-mathjax-preview": "2.2.6"

**Review by awais-ansari — APPROVED** [review]

**Review by muhammadadeeltajamul — APPROVED** [review]

**Comment by xitij2000** [comment]
@mehaknasir @awais-ansari MathType, the plugin used for TinyMCE here doesn't seem to be free, it has a limited use licence, and additionally a lot of the UI is being loaded from a third party site. Additionally, anything typed in the math editor is sent to a third-party site where it's rendered to an image. 

It also sends analytics data to the third party wiris.net site. Is all of that OK?

**Comment by mehaknasir** [comment]
> @mehaknasir @awais-ansari MathType, the plugin used for TinyMCE here doesn't seem to be free, it has a limited use license, and additionally, a lot of the UI is being loaded from a third-party site. Additionally, anything typed in the math editor is sent to a third-party site where it's rendered to an image.
> 
> It also sends analytics data to the third-party wiris.net site. Is all of that OK?

@xitij2000 thank you for raising these points, I will discuss that. about the availability of these plugins, I was sure that were free, but I will check it again.

**Comment by mehaknasir** [comment]
@xitij2000 also can yo please provide the source of information you just listed above? Thanks.

**Pull request merged**